documentation/operators/src/nodes/troubleshooting.md

```diff
@@ -308,10 +291,25 @@ Finally, restart your node. You should see if the mix node is using the port you
 
 ### What is `verloc` and do I have to configure my mix node to implement it?
 
-`verloc` is short for _verifiable location_. Mixnodes and gateways now measure speed-of-light distances to each other, in an attempt to verify how far apart they are. In later releases, this will allow us to algorithmically verify node locations in a non-fakeable and trustworthy manner.
+`verloc` is short for _verifiable location_. Mixnodes and gateways now measure speed-of-light distances to each other, in an attempt to verify how far apart they are. In later releases, this will allow us to algorithmically verify node locations in a non-fake-able and trustworthy manner.
 
 You don't have to do any additional configuration for your node to implement this, it is a passive process that runs in the background of the mixnet from version `0.10.1` onward.
 
+## Gateways & Network requesters
+
+### My gateway seems to be running but appears offline
+
+Check if your [firewall status](./maintenance.md#configure-your-firewall) is active and if the needed ports are allowed.
```

Comment:
small grammatical change: 

"Check your firewall is active and if the necessary ports are open / allowed"

Reply:
done